nitro-protocol/contracts/ForceMove.sol

```diff
@@ -157,10 +157,26 @@ contract ForceMove is IForceMove, StatusManager {
             'Signer not authorized mover'
         );
 
-        _requireValidTransition(
-            fixedPart.participants.length,
-            variablePartAB,
-            fixedPart.appDefinition
+        VariablePart[] memory variableParts = new VariablePart[](2);
+        variableParts[0] = variablePartAB[0];
+        variableParts[1] = variablePartAB[1];
```

Comment:
I think we may be able to remove `respond` from the on chain API.  It seems closely coupled to the idea of a valid transition, which we are deprecating.

Reply:
Do you suggest `checkpoint` being used instead of `respond`?

Reply:
Exactly!